tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs

```diff
@@ -23,10 +24,17 @@ internal static class WafConfigurator
 
         internal static InitializationResult Configure(string rulesFile, WafNative wafNative, Encoder encoder, string obfuscationParameterKeyRegex, string obfuscationParameterValueRegex)
         {
+            if (wafNative.ExportErrorHappened)
```

Comment:
Doesn't need to be this PR, but I wonder, can/should we add this to telemetry/configuration log

Reply:
It's a good idea, we already have the WAF version in the configuration log, so adding this should be straight forward.

I noticed there's quite a lot of stuff that's in the config log, but not startup telemetry, I guess we could add the missing stuff as "additional values" ?

Reply:
I added this error in the startup log if it's `true` .  As we want to have it for next release, I suggest we do a subsequent PR on adding security info to startup telemetry ( `ConfigurationTelemetryCollector` for me to remember 😅 )